src/infiniteGrid.js

```diff
@@ -661,8 +662,8 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document], function($, ns, glob
 			$.each(needCheck, function(i, v) {
 				$el = $(v);
 
-				// for IE10 lower
-				if (self._isIE10lower) {
+				// workaround for IE10
```

Comment:
The comment is quite confusing, because condition is applied on every IE environment.